apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brev/detaljer/MottakerPanel.tsx

```diff
@@ -26,101 +27,121 @@ export default function MottakerPanel({
   const soekerErIkkeMottaker = soekerFnr !== mottaker.foedselsnummer?.value
 
   return (
-    <Panel border>
-      <Heading spacing level="2" size="medium">
-        Mottaker
-        {redigerbar && <RedigerMottakerModal brev={vedtaksbrev} oppdater={oppdater} vergeadresse={vergeadresse} />}
-      </Heading>
-
-      {soekerErIkkeMottaker && (
-        <Alert variant="info" size="small" inline>
-          Søker er ikke mottaker av brevet
-        </Alert>
-      )}
-      {vergeadresse && (
-        <Alert variant="info" size="small" inline>
-          Søker har verge
-        </Alert>
-      )}
-      <br />
+    <>
+      <Panel border>
+        <FlexRow justify="space-between">
+          <Heading spacing level="2" size="medium">
+            Mottaker
+          </Heading>
+          <div>
+            {redigerbar && (
+              <Button
+                variant="secondary"
+                onClick={() => setErModalAapen(true)}
+                icon={<DocPencilIcon aria-hidden />}
+                size="small"
+              />
+            )}
+          </div>
+        </FlexRow>
 
-      <InfoWrapper>
-        <Info
-          wide
-          label="Navn"
-          tekst={
-            /[a-zA-Z\s]/.test(mottaker.navn) ? (
-              mottaker.navn
-            ) : (
-              <Alert variant="error" size="small" inline>
-                Navn mangler
-              </Alert>
-            )
-          }
-        />
-        {mottaker.foedselsnummer && <Info label="Fødselsnummer" tekst={mottaker.foedselsnummer.value} wide />}
-        {mottaker.orgnummer && <Info label="Org.nr." tekst={mottaker.orgnummer} wide />}
+        {soekerErIkkeMottaker && (
+          <Alert variant="info" size="small" inline>
+            Søker er ikke mottaker av brevet
+          </Alert>
+        )}
+        {vergeadresse && (
+          <Alert variant="info" size="small" inline>
+            Søker har verge
+          </Alert>
+        )}
+        <br />
 
-        <Info
-          wide
-          label="Adresse"
-          tekst={
-            <>
-              {!adresse?.adresselinje1 && !adresse?.adresselinje2 && !adresse?.adresselinje3 ? (
-                <Alert variant="warning" size="small" inline>
-                  Adresselinjer mangler
-                </Alert>
+        <InfoWrapper>
+          <Info
+            wide
+            label="Navn"
+            tekst={
+              /[a-zA-Z\s]/.test(mottaker.navn) ? (
```

Comment:
Denne regex'en virker megasuspekt -- den aksepterer alle strenger som inneholder en bokstav a-z eller whitespace. Det kan umulig gi mer mening enn `mottaker.navn.length > 0` f.eks., men jeg vet ikke hva som originalt er tenkt her (ikke at du har introdusert regex'en i denne PR'en da)

Reply:
Lar denne bli for nå, tror det er Nicolas som har styringen på den, men han er på ferie